src/index/script/CanvasPosition/_CanvasPosition_addPos.test.ts

```diff
@@ -0,0 +1,29 @@
+import Point from "../../../_common/script/Point";
+import CanvasPosition from "./CanvasPosition";
+import CanvasPosition_setPos from "./_CanvasPosition_setPos.test";
+
+export default class CanvasPosition_addPos extends CanvasPosition_setPos {
+	
+	protected buildOneTestCaseAndExpected(i: number): [any, any] {
```

Comment:
I would encourage you to use more descriptive names for your tests such as 'validateTwoPointsShouldEqualEachOther' or something that another dev can look at and go 'ahh' I get the intent of this test case.  I know ugly name - but it does communicate a message to another dev.

Reply:
I think a re-visit of how I do testing is in order. I probably need a `UnitTestByScattershot` base class, since that's how most of my tests are run; I genereate a ton of random expected outputs, derive corresponding inputs, then fire those derived inputs through the testing function. The base class a unit test derives from should tell the reader what type of test they're looking at.